src/witan/send/chart.clj

```diff
@@ -387,9 +390,32 @@
 (defn save-workbook [file-name wb]
   (xl/save-workbook! file-name wb))
 
-(defn insert-zero-counts [census-data counts]
-  (mapcat (fn [cy] (map (fn [ay] (if-let [ay-map (some (fn [m] (when (#{ay} (:academic-year m)) m)) (val cy))]
-                                   ay-map
-                                   {:calendar-year (:calendar-year (-> cy val first)), :academic-year ay, :population 0}))
-                        (into (sorted-set) (map :academic-year census-data))))
-          (group-by :calendar-year counts)))
+(defn insert-zero-counts
+  ;; FIXME: This assumes it is for academic year.
+  ([census-data counts]
```

Comment:
Do we want to keep the two arity version if it makes some assumptions?

Reply:
no we don't, but I didn't want to break anyone's code that was using it. :-)

Reply:
good call